lib/logout.js

```diff
@@ -350,4 +351,30 @@ function prepareAndSendToken(req, res, element_type, token, options, cb) {
 
     send(params);
   });
-}
\ No newline at end of file
+}
+
+module.exports.logoutError = function(options){
+  return function(req, res, next){
+    if (!options.destination) {
+      return next(new Error('Destination not specified'));
+    }
+
+    var error = options.error || {};
+    
+    var logoutResponse = templates.logoutresponse({
+      id:             '_' + utils.generateUniqueID(),
+      instant:        utils.generateInstant(),
+      inResponseTo:   options.id,
+      destination:    options.destination,
+      issuer:         options.issuer,
+      samlStatusCode:  error.code || STATUS.RESPONDER,
+      samlStatusMessage: error.description
+    });
+
+    // Signature location
+    options.reference = constants.ELEMENTS.LOGOUT_RESPONSE.SIGNATURE_LOCATION_PATH;
+    options.relayState = options.relayState || req.query.RelayState || (req.body || {}).RelayState || '';
```

Comment:
`|| ''` not needed, `prepareAndSendToken` does it https://github.com/auth0/node-samlp/blob/df1982e/lib/logout.js#L318